### PR TITLE
Open edX Release Notes

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@singingwolfboy @nedbat @sarina @mollydb   long wait for RTD build.  main file to review is Birch.rst

@srpearce  can you proof that file

File: en_us/open_edx_release_notes /source/birch.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
+####################################
+Open edX Birch Release
+####################################
+
+The Birch release of Open edX Platform includes many new features, from new
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: of _the_ Open edX

File: en_us/open_edx_release_notes /source/birch.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+can be used in randomized assignments.
+
+For more information, see `Working with Content Libraries`_ in the `Building and Running an Open edX Course`_ guide.
+
+********************
+Prerequisite Courses
+********************
+
+You can now require that students pass a specific  course before they enroll in
+your course.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst:49]: Extra space between "specific" and "course"

File: en_us/open_edx_release_notes /source/birch.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+For more information, see `Specify Prerequisite Courses`_ in the *Building and
+Running an Open edX Course* guide.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Add a link to the guide (like the one in line 42)

File: en_us/open_edx_release_notes /source/birch.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+
+********************
+Entrance Exams
+********************
+
+You can now require that students pass an entrance exam before they access your course materials.
+
+For more information, see `Require an Entrance Exam`_ in the *Building and
+Running an Open edX Course* guide.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Add a link to the guide (like the one in line 42)

File: en_us/open_edx_release_notes /source/birch.rst [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+******************** 
+Student Notes
+********************
+
+A learner can highlight text and make notes while progressing through a course.
+The learner can then review his notes in the body of the course or on a
+separate Notes tab.
+
+For more information, see `Student Notes Tool`_ in the *Building and
+Running an Open edX Course* guide.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Add a link to the guide (like the one in line 42)

File: en_us/open_edx_release_notes /source/birch.rst [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+********************
+Course Reruns
+********************
+
+You can create a new course easily by re-running an existing course. When you
+re-run a course, most, but not all, of the original course content is
+duplicated to the new course. 
+
+For more information, see `Rerunning a Course`_ in the *Building and
+Running an Open edX Course* guide.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Add a link to the guide (like the one in line 42)

File: en_us/open_edx_release_notes /source/birch.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+For more information, see `Student Notes Tool`_ in the *Building and
+Running an Open edX Course* guide.
+
+********************
+Course Reruns
+********************
+
+You can create a new course easily by re-running an existing course. When you
+re-run a course, most, but not all, of the original course content is
+duplicated to the new course.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: ... is copied into the new course

File: en_us/open_edx_release_notes /source/birch.rst [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+For more information, see `Rerunning a Course`_ in the *Building and
+Running an Open edX Course* guide.
+
+*******************************************
+Google Calendar and Google Drive Components
+*******************************************
+
+You can embed Google calendars and Google Drive files in your course. Learners
+see the calendar or file directly in the courseware. Learners can also interact
+with Google Forms files, completing the form or survey in the courseware.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: ... completing forms or surveys

File: en_us/open_edx_release_notes /source/birch.rst [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+For more information, see `Google Calendar Tool`_ and `Google Drive Files
+Tool`_ in the `Building and Running an Open edX Course`_ guide.
+ 
+************************************************** 
+Support for Graded Problems in Content Experiments
+**************************************************
+
+You can now use graded problems inside content experiments.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Content experiments can now include graded problems.

File: en_us/open_edx_release_notes /source/birch.rst [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+* `Enrollment API`_
+* `Data Analytics API`_
+
+
+************** 
+Other Changes
+**************
+
+In addition to the new features listed above, the Birch release includes many
+small changes and bug fixes. See the edX Release Notes for changes listed after
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/birch.rst]: Link to release notes?

File: en_us/open_edx_release_notes /source/index.rst [hunk 11]
```diff
@@ -0,0 +1,32 @@
+####################################
+Open edX Release Notes
+####################################
+
+The *Open edX Release Notes* provide information about the Open edX named
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/index.rst:5]: *Release (decap the L)

File: en_us/open_edx_release_notes /source/index.rst [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+   read_me
+   preface
+   browsers
+   birch
+   
+**************************
+Open edX Documentation
+**************************
+
+All edX Documentation is available at `docs.edx.org`_.
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/index.rst:23]: Decap "Documentation"

File: en_us/open_edx_release_notes /source/read_me.rst [hunk 13]
```diff
@@ -0,0 +1,15 @@
+*******
+Read Me
+*******
+
+This document is created using RST_ files and Sphinx_. You, the user community,
+can help update and revise this documentation project on GitHub::
```
Comment (srpearce) [inline en_us/open_edx_release_notes /source/read_me.rst:6]: Double colon?

**Comment by srpearce** [comment]
Copy editing pass complete.

**Pull request merged**